packages/charts/src/chart_types/heatmap/specs/heatmap.ts

```diff
@@ -22,7 +22,8 @@ import { config } from '../layout/config/config';
 import { Config } from '../layout/types/config_types';
 import { X_SCALE_DEFAULT } from './scale_defaults';
 
-const defaultProps = {
+/** @internal */
+export const defaultProps = {
```

Comment:
No need to export this

```suggestion
const defaultProps = {
```

Reply:
Fixed in eb9c5ab thanks!